src/ng/directive/ngOptions.js

```diff
@@ -147,37 +147,39 @@ var ngOptionsMinErr = minErr('ngOptions');
         <div ng-controller="ExampleController">
           <ul>
             <li ng-repeat="color in colors">
-              Name: <input ng-model="color.name">
-              <input type="checkbox" ng-model="color.notAnOption"> Disabled?
-              [<a href ng-click="colors.splice($index, 1)">X</a>]
+              <label>Name: <input ng-model="color.name"></label>
+              <label><input type="checkbox" ng-model="color.notAnOption"> Disabled?</label>
+              [<button ng-click="colors.splice($index, 1)" aria-label="remove">X</button>]
             </li>
             <li>
-              [<a href ng-click="colors.push({})">add</a>]
+              [<button ng-click="colors.push({})">add</button>]
             </li>
           </ul>
           <hr/>
-          Color (null not allowed):
-          <select ng-model="myColor" ng-options="color.name for color in colors"></select><br>
-
-          Color (null allowed):
+          <label>Color (null not allowed):
+            <select ng-model="myColor" ng-options="color.name for color in colors"></select><br>
+          </label>
+          <label>Color (null allowed):
           <span  class="nullable">
             <select ng-model="myColor" ng-options="color.name for color in colors">
               <option value="">-- choose color --</option>
             </select>
-          </span><br/>
+          </span></label><br/>
 
-          Color grouped by shade:
-          <select ng-model="myColor" ng-options="color.name group by color.shade for color in colors">
-          </select><br/>
+          <label>Color grouped by shade:
+            <select ng-model="myColor" ng-options="color.name group by color.shade for color in colors">
+            </select>
+          </label><br/>
 
-          Color grouped by shade, with some disabled:
-          <select ng-model="myColor"
-                  ng-options="color.name group by color.shade disable when color.notAnOption for color in colors">
-          </select><br/>
+          <label>Color grouped by shade, with some disabled:
+            <select ng-model="myColor"
+                    ng-options="color.name group by color.shade disable when color.notAnOption for color in colors">
```

Comment:
At least in Chrome, you can break strings up with no problem. But since this is one long string I believe it's OK even if it's a little longer than 100 chars.